install.sh

```diff
@@ -0,0 +1,68 @@
+
+#n to check if a package is installed
+package_installed() {
+    package="$1"
+    dpkg -s "$package" >/dev/null 2>&1
+}
+
+# Check if Vim is installed
+if ! package_installed "vim"; then
+    echo "Installing Vim..."
+    apt-get update
+    apt-get install -y vim
+else
+    echo "Vim is already installed."
+fi
+
+# Check if Tmux is installed
+if ! package_installed "tmux"; then
+    echo "Installing Tmux..."
+    apt-get update
+    apt-get install -y tmux
+else
+    echo "Tmux is already installed."
+fi
+
+# Check if Git is installed
+if ! package_installed "git"; then
+    echo "Installing Git..."
+    apt-get update
+    apt-get install -y git
+else
+    echo "Git is already installed."
+fi
+
+# Set the paths for your configuration files
+nbrc_path="$HOME/.dotfiles/nbrc"
+tmuxrc_path="$HOME/.dotfiles/tmuxrc"
+vimrc_path="$HOME/.dotfiles/vimrc"
+gitconfig_path="$HOME/.dotfiles/gitconfig"
+gitinstall_path="$HOME/.dotfiles/gitconfig/gitinstall.sh"
```

Comment:
NOTE: This is a bash script. Why do you need to make a symlink out of it?
I believe you have to run your script from within this file.

Reply:
how even in your dotfile you made a symlink for vimrc and tmux and i didnt make a symlink for gitconfig